doc/using_the_api.md

```diff
@@ -222,6 +222,19 @@ All endpoints are prefixed with `/api/`
     ]
 ```
 
+#### ENDPOINT: USER INVITE
+
+| method | route           | action                 |
+| :----- | :-------------- | :--------------------- |
+| POST   | `/user/invite`  | Invite a user          |
+
+##### Response (200)
```

Comment:
As of backend PR #201, we'll be returning a 201 response from this endpoint.  Let's go ahead and reflect that change here. 
```suggestion
##### Response (201)
```